src/styles/field.styl

```diff
@@ -74,3 +74,8 @@ text-button-color = #297ef1
 
             label
                 color pomegranate
+
+        .coz-field-input
+            /* input in cozy-ui is inline-block with a flex property */
+            flex    none!important
```

Comment:
We'll have to find a way to remove this `!important` to pass the linter. And to stay proud of ourselves.

Reply:
Oh... there's no other way here. And I just don't understand why our inputs is flex. Ping @GoOz 
I think I will have to change the lint stylus here :/

Reply:
There is always a path ... what about:
```stylus
input.coz-field-input
  flex: none
```
Or if it not enough
```stylus
input.coz-field-input[type=text],
input.coz-field-input[type=password]
  flex: none
```
What does @GoOz think ?

Reply:
The flex property was from collect, my bad...
So it's better now

Reply:
There is always a way to avoid an `!important` 😅